ietf-te-packet-types.yang

```diff
@@ -767,17 +759,93 @@ module ietf-te-packet-types {
     }
   }
 
+  // CHANGE NOTE: The te-packet-path-bandwidth below has been
+  // added in this module revision
+  // RFC Editor: remove the note above and this note
+  grouping te-packet-path-bandwidth {
+    description
+      "Bandwidth attributes for TE Packet paths.";
+    container packet-bandwidth {
+      description
+        "Bandwidth attributes for TE Packet paths.";
+      leaf specification-type {
+        type te-bandwidth-requested-type;
+        description
+          "The method used for setting the bandwidth, either
+           explicitly specified or automatically computed.";
+      }
+      leaf set-bandwidth {
+        when "../specification-type = 'specified-value'" {
+          description
+            "When the bandwidth value is explicitly specified.";
+        }
+        type bandwidth-kbps;
+        description
+          "Set the bandwidth value explicitly, e.g., using offline
+           calculation.";
+      }
+      container bandwidth-profile {
+        when "../specification-type = 'specified-profile'" {
+          description
+            "When the bandwidth profile is explicitly specified.";
+        }
+        description
+          "Set the bandwidth profile attributes explicitly.";
+        leaf bandwidth-profile-name {
+          type string;
+          description
+            "Name of Bandwidth Profile.";
+        }
+        leaf bandwidth-profile-type {
+          type identityref {
+            base bandwidth-profile-type;
+          }
+          description
+            "Type of Bandwidth Profile.";
+        }
+        uses bandwidth-profile-parameters;
+      }
+      leaf class-type {
+        type te-types:te-ds-class;
+        description
+          "The Class-Type of traffic transported by the LSP.";
+        reference
+          "RFC 4124: Protocol Extensions for Support of
+                     Diffserv-aware MPLS Traffic Engineering,
+                     Section 4.3.1";
+      }
+      container state {
```

Comment:
do we still use "state" container? Also, shouldn't this be config False?

Reply:
Agreed.

See also: https://github.com/tsaad-dev/te/pull/291#issuecomment-2379389176

Reply:
Fixed in https://github.com/tsaad-dev/te/pull/291/commits/4b4632c6c61ab00d260a5171e979af5143999c54